easybuild/easyconfigs/f/freebayes/freebayes-1.1.0-goolf-1.7.20_39e5e4b.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'freebayes'
+version = '1.1.0'
+versionsuffix = '_39e5e4b'
+
+homepage = 'https://github.com/ekg/freebayes'
+description = """ Bayesian haplotype-based polymorphism discovery and genotyping. """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no direct download url. As described in documentation the download has been done with
+# $> git clone --recursive git://github.com/ekg/freebayes.git
+# Then compressed and placed in the sources folder with name "freebayes-1.1.0_39e5e4b.tar.gz"
+sources = ['%(name)s-%(version)s%(versionsuffix)s.tar.gz']
```

Comment:
Urgh, OK... then mention that in the comment, why you *must* use `git clone --recursive`?

I'm not sure if we should include the commit ID in the `versionsuffix` though, since the mapping from version to commit is very clear via https://github.com/ekg/freebayes/releases?